### ISSUE TITLE
(1.20.4)(Quilt) Crashing to title screen when loading singleplayer

### ISSUE THREAD

**Issue by TheRealDanini** [ReplayMod/ReplayMod#959]
![Screenshot_20240507_141144](https://github.com/ReplayMod/ReplayMod/assets/126975233/616e74a2-f5ee-4644-8dea-8dee5f5728df)
Getting this error when trying to load singleplayer with the latest replaymod on quilt 1.20.4

Steps to reproduce

- Create an instance of minecraft 1.20.4 fabric/quilt and install replaymod
- Create/Join into any singleplayer world
- The game should crash with the above error

Expected result - Minecraft should proceed as normal

Actual result - Game crashes to title screen with mentioned error

Log 
[2024-05-07-7.log](https://github.com/ReplayMod/ReplayMod/files/15239433/2024-05-07-7.log)

**Comment by haaldor** [comment]
I am getting the same error on 1.20.2, however after doing MRE I found out it was combination of Sodium (version 0.5.5) and Replay mod.
It's not obvoius when downloading mod from Modrinth, but when reading website/curse forge it's clear it's incompatible.

**Comment by TheRealDanini** [comment]
Yeah, I download most mods from modrinth so I would not have known Sodium 0.5.5 would have caused the issue. Guess we will have to wait for a patch to come out to fix it if it can be fixed.
